Sources/XCRemoteCache/Config/XCRemoteCacheConfig.swift

```diff
@@ -83,6 +83,8 @@ public struct XCRemoteCacheConfig: Encodable {
     var downloadRetries: Int = 0
     /// Number of retries for upload requests
     var uploadRetries: Int = 3
+    /// Delay between retries
+    var retryDelay: Double = 10.0
```

Comment:
Nice to have it as a config, :+1:
Can we also document that in .md?

Reply:
Thx, fixed